Modules/Core/Common/wrapping/itkAnatomicalOrientation.wrap

```diff
@@ -1 +1,7 @@
 itk_wrap_simple_class("itk::AnatomicalOrientation")
+
+
+set(WRAPPER_AUTO_INCLUDE_HEADERS OFF)
+itk_wrap_simple_class("itk::AnatomicalOrientation::PositiveEnum")
+itk_wrap_simple_class("itk::AnatomicalOrientation::NegativeEnum")
+itk_wrap_simple_class("itk::AnatomicalOrientation::CoordinateEnum")
```

Comment:
@thewtex Here is the latest compiler errors: 

[Wrapping/Modules/ITKCommon/itkAnatomicalOrientationPython.cpp:3506](https://github.com/InsightSoftwareConsortium/ITK/blob/e066e3372ca4032e3dd6e92b0ba1003dddbf19d0/Wrapping/Modules/ITKCommon/itkAnatomicalOrientationPython.cpp#L3506):24: error: expected namespace name
 using namespace ::itk::AnatomicalOrientation;

I have tried a couple different things. But it appeared that the enum inside the AnatomicalOrientation class were not getting wrapped because the OrientImageFilterTest.py test was failing.